.github/workflows/run_setup.yaml

```diff
@@ -9,6 +9,9 @@ on:
     branches: [ main, master ]
   pull_request:
     branches: [ main, master ]
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1"
```

Comment:
Maybe add some random hour and minute in the middle of the night to relieve stress on build servers?

Reply:
Jenkins has an option that allows the server to determine a start a build in a timeframe according to its needs (with `H H(0-1) * * * `), do you know whether that's possible here as well? I did not find anything

otherwise, we would want to have the same time for all workflows, to have the possible fail messages arrive at the same time, right?

Reply:
I also did not find anything - I think we should simply use a random time consistently for all actions.

Reply:
done with 33f4f3b